docs/release-notes/2.42.md

```diff
@@ -0,0 +1,122 @@
+- setting the limit too high can impact performance.
+- if no limit is specified or the option is disabled, the server will manage the connection limits automatically.
+
+### PMM Agent
+
+Similarly, you can now limit the number of connections that the PMM Agent opens to monitored databases for QAN, Advisors, and other systems.
+
+Starting with this release, pmm-agent tracks all required resources (primarily connections), and opens no more than two connections per database instance simultaneously.
+
+You can change this default value by setting a parameter or environment variable when starting the pmm-agennt.
+
+## Experimental PMM Self-monitoring dashboard
+
+We've added a new experimental **PMM Health** dashboard to provide detailed insights into the health and performance of PMM itself.
+
+The dashboard reflects the PMM architecture and covers key components such as overall component status, node health, Query Analytics, PMM-managed, Grafana, VictoriaMetrics, ClickHouse, and PostgreSQL metrics.
+
+This initial version is available in the **Experimental** folder after updating PMM and contains a starter set of key metrics to help identify potential issues and ensure optimal operation.
+
+For more information about this new dashboard, see the [Keeping an eye on the eye blog post]().
+
+![!image](../_images/PMM_Health_dashboard.png)
+
+## Experimental MongoDB dashboards: Sharded Cluster and Replica Set
+
+Along with the PMM Health Dashboard, we're also introducing experimental updates to two key MongoDB dashboards: the **Cluster Summary** and **ReplicaSet** dashboards. These new versions, accessible from the MongoDB section within PMM, have been redesigned to address feedback regarding readability and data density.
+
+The design and navigation are simplified here, to focus on the essential parameters and metrics most relevant to MongoDB performance monitoring.
+
+We encourage you to explore these experimental dashboards and provide feedback in the PMM forum to help us refine them.
+
+![!image](../_images/MongoDB_Cluster_Summary.png)
+
+![!image](../_images/ReplSet_Dashboard_New.png)
+
+### Improvements
+
+- [PMM-3303](https://perconadev.atlassian.net/browse/PMM-3303) - We've introduced an experimental PMM Health dashboard in the Experimental folder, offering detailed insights into PMM's health and performance by covering key components such as node health, Query Analytics, Grafana, VictoriaMetrics, ClickHouse, and PostgreSQL metrics.
+
+- [PMM-13123](https://perconadev.atlassian.net/browse/PMM-13123) - Enhanced the **Query Response Time Distribution** graphs in the **MySQL Query Response Time Details** dashboard to provide more granular insights into query performance. Previously, the graphs did not include buckets for queries with response times less than 100 milliseconds, which is where many queries typically fall. This limitation made it difficult to analyze the performance of sub-100ms queries, especially those in the sub-1ms range.
```

Comment:
```suggestion
- [PMM-13123](https://perconadev.atlassian.net/browse/PMM-13123) - Enhanced the **Query Response Time Distribution** graphs in the **MySQL Query Response Time Details** dashboard to provide more granular insights into query performance. Previously, the graphs did not include buckets for queries with response times less than 100 milliseconds, which is where many queries typically fail. This limitation made it difficult to analyze the performance of sub-100ms queries, especially those in the sub-1ms range.
```
Not sure about this one, could as well be `fall` :)

Reply:
In this context, many queries have response times below 100 milliseconds, so they "fall into" that range..

Reply:
Shall we then say `...fall into`?

Reply:
Indeed, I've reworded it to clarify: 

- [PMM-13123](https://perconadev.atlassian.net/browse/PMM-13123) - Enhanced the **Query Response Time Distribution** graphs in the **MySQL Query Response Time Details** dashboard to provide more granular insights into query performance. Previously, the graphs did not include buckets for queries with response times less than 100 milliseconds, the **range which many queries typically fall into.** This limitation made it difficult to analyze the performance of sub-100ms queries, especially those in the sub-1ms range.